### PR TITLE
Prevent SE to index some pages 

### PR DESCRIPTION
**Pull request by cbeauchesne**
#2101

### CODE REVIEW

File: src/js/vue-plugins/router.js [hunk 1]
```diff
@@ -176,4 +196,41 @@ const router = new Router({
...
+  const nearestWithMeta = to.matched
+    .slice()
+    .reverse()
+    .find((r) => r.meta && r.meta.tags);
+
+  // Remove any stale meta tags from the document using the key attribute we set below.
+  Array.from(document.querySelectorAll('[data-vue-router-controlled]')).map((el) => el.parentNode.removeChild(el));
+
+  // Skip rendering meta tags if there are none.
+  if (!nearestWithMeta) return next();
```
Comment (brunobesson) [inline src/js/vue-plugins/router.js:214]: please always use brackets around ifs (I should set a linter rule)

File: src/js/vue-plugins/router.js [hunk 2]
```diff
@@ -176,4 +196,41 @@ const router = new Router({
   },
 });
 
+router.beforeEach((to, from, next) => {
```
Comment (brunobesson) [inline src/js/vue-plugins/router.js:199]: I wonder if this could somehow be handled through vue-meta which we already use?

**Review by brunobesson — APPROVED** [review]

File: src/js/vue-plugins/router.js (hunk 2 above)
Comment (cbeauchesne) [reply src/js/vue-plugins/router.js:199]: Definitly !!! Actually, we use `vue-head`. 

I looked for vue-meta, and as I did not found it, I did this small code (didn't want to add new dep).

Let me update this.

**Comment by cbeauchesne** [comment]
@brunobesson I also fixed a small bug. Go from one waypoint to another one, you will see that meta are added, not updated. There are other issue with the same root cause. See here : https://github.com/ktquez/vue-head#replace-content-the-elements

The solution is simply add `id` on unique meta tags. 

Could you redo a review please ?

**Review by brunobesson — APPROVED** [review]

**Pull request merged**